dvc/command/init.py

```diff
@@ -56,4 +59,13 @@ def add_parser(subparsers, parent_parser):
             "This operation removes local cache."
         ),
     )
+    init_parser.add_argument(
+        "--subrepo",
```

Comment:
I read #2349 again and I'm no longer sure this is an appropriate name. Are we talking about running `dvc init` in subdirectories of a Git repo, which are NOT Git subrepos themselves (no `git init` is run in them) ?

If that's the case and they are not Git subrepos (just subdirs), then `--subrepo` seems quite confusing. Maybe `--subdir`, `--subproject`, or `--subdvc`.

Reply:
I would go for `--subdir` cc @efiop @Suor

Reply:
@jorgeorpinel `--subdir` sounds good to me too! @pared What do you think?

Reply:
I like it, ill refactor.